AirCasting/StreamSynchronizer/StreamRemover.swift

```diff
@@ -0,0 +1,90 @@
+// Created by Lunar on 22/11/2021.
+//
+
+import Foundation
+
+protocol StreamRemover {
+    func deleteStreams(client: APIClient, session: SessionEntity, completion: @escaping () -> Void)
+}
+
+class StreamRemoverDefault: StreamRemover {
+
```

Comment:
I think we ditched spaces after class? (not sure tho)

Reply:
I think we do still have them. It looks better, isn't it?

Reply:
Matter of taste, but some can get really religious about this sort of details (like tabs vs spaces ;) ). Personally I don't like empty lines after class keyword, but won't try to force my opinion here, I think majority of the team likes them so I'll adjust